packages/lit-dev-content/.eleventy.js

```diff
@@ -77,6 +77,9 @@ module.exports = function (eleventyConfig) {
     '../lit-dev-content/samples/tutorials/**/tutorial.json'
   );
   eleventyConfig.addWatchTarget('../lit-dev-content/samples/tutorials/**/*.md');
+  for (const component of componentsToSSR) {
```

Comment:
Tiny nit, does the `ssr.js` file need a license header?

```
/**
 * @license
 * Copyright 2022 Google LLC
 * SPDX-License-Identifier: BSD-3-Clause
 */
```

Just so I understand - this is adding watch for dev-mode for the SSR'd components.